lms/djangoapps/course_api/serializers.py

```diff
@@ -112,10 +115,15 @@ def get_data_with_marketing_overrides(self, original_serialized_data):
         overridden = {}
         overridden.update(original_serialized_data)
 
-        overridden['effort'] = marketing_data['estimated_effort']
+        overridden['effort'] = marketing_data['effort']
```

Comment:
@shadinaif I don't see the related changes for https://github.com/Edraak/edx-platform/pull/309 . `MarketingCourseDetailSerializer` is completely missing, right?

Reply:
@OmarEdraak it has been refactored earlier in https://github.com/Edraak/edraak-platform/pull/59
the name now is `CourseDetailMarketingSerializer` and inherited from `CourseSerializer` instead of `CourseDetailSerializer`

Also, in the same PR https://github.com/Edraak/edraak-platform/pull/59 changed the behaviour to dynamically set the serializer here https://github.com/Edraak/edraak-platform/blob/2e925561d2d463e84193fc3c8ccd4cb44e158a97/lms/djangoapps/course_api/views.py#L103

Do you have any suggestion before we merge?

Reply:
Ok, it's all just rust code :sweat_smile: let's get it merged. We'll have a better "code review" by integrating all the pieces together. Then we can fix bugs if any.

Reply:
Agree :+1: